lib/parse_simplify_options.js

```diff
@@ -1,4 +1,6 @@
 const propsMap = {
+  values: 'keepRichValues',
```

Comment:
`values` is a bit ambiguous, as the simplify object already as a `value` attribute without the rich values being kept; what about
```suggestion
  richvalues: 'keepRichValues',
```